index.html

```diff
@@ -14,13 +14,14 @@
         <p id="killerName"></p>
```

Comment:
Also the first article tag is missing for your killer card, but I can see you do have the end one. If you refer to the weapon deck you can see that it has it's start and end article tag. `<article class="weapon-deck" id="weaponCard"> <p>WEAPON</p>  <p id="weaponName"></p> <p id="weaponWeight"></p> </article>`